src/index.html

```diff
@@ -14,3 +15,9 @@
 </body>
 
 </html>
+
+<style>
+  body {
+    background-image:  url("assets/pictures/background.jpg");
+  }
+</style>
```

Comment:
did you intend for this style block to be added here?

Reply:
yes

Reply:
typically, style blocks are placed in between the `<head>` tag. could you explain why they're outside of the `html` tags? this is not a typical way of using the `style` block so im curious why you've done it this way